democracylab/settings.py

```diff
@@ -90,6 +90,8 @@
     },
 }
 
+GITHUB_API_TOKEN = os.environ.get('GITHUB_API_TOKEN', None)
```

Comment:
I'd suggest adding a warning to ENVIRONMENT_VARIABLE_WARNINGS for this key about the rate limiter so (for example) a developer will know the github pulls may not work properly if it's not set in a dev environment.